distro/configuration/concepts/drugs_concepts.csv

```diff
@@ -1,14 +1,12 @@
 Uuid,Void/Retire,Short name:en,Fully specified name:en,Fully specified name:fr,Data class,Data Type,Members,_version:1,_order:500
```

Comment:
Ditto here as for drugs: the concepts need to be fixed. We shouldn't be removing them.

Reply:
Removing them sounds like the right fix since they are duplicates already brought in by the OCL packages.

Reply:
Ok... As long as their all duplicated.